docs/querying/sql-functions.md

```diff
@@ -1103,11 +1124,32 @@ Returns the following:
 
 ## EXTRACT
 
-`EXTRACT(<TIME_UNIT> FROM <TIMESTAMP>)`
+Extracts the value of some unit of the timestamp. 
```

Comment:
Should this say "from the timestamp"?

Reply:
Extracts the value of a unit from the timestamp